server/intelligence-service/app/mentor/persona_prompt.py

```diff
@@ -0,0 +1,34 @@
+persona_prompt = """
```

Comment:
Storing these large prompts in `.py`-files doesn't seem like an efficient solution. It's probably fine for this one prompt, but we should have a more modular system in place for when we need more prompts (especially interesting for versioning).

My suggestion would be:
- have a dedicated file format for prompts (`.yml` / `.md` / `.txt`), whatever fits
- dedicated prompt loader class

I asked Claude to provide an example implementation and got this:
https://gist.github.com/GODrums/a391ac8cdef77fbb0f7397ed1da3e87d

There are existing solutions for this (e.g. https://github.com/deadbits/prompt-serve), but I think we can easily implement this ourselves instead.

Reply:
@GODrums very vers good point! Sure, the current implementation is not scalable 😅
I also wouldn't use an extra-solution for that as of now. `.txt` files are currently the way to go in my opinion.

I have left the existing prompts for the greeting and overall bot interaction in the same file for now as I am going to re-structure those one more time anyways in the following PR.

Reply:
Sounds great to me.

I just checked out your changes in 13be91e and you got a small typo in the new folder name for prompts: 
```
server/intelligence-service/app/mentor/propmpts/mentor_persona.txt
```